paddle/fluid/inference/api/analysis_predictor.h

```diff
@@ -141,6 +144,14 @@ class AnalysisPredictor : public PaddlePredictor {
   std::vector<framework::OpDesc *> fetches_;
   std::map<size_t, std::string> idx2fetches_;
 
+#if PADDLE_WITH_MKLDNN
+  // Helper class to perform quantization
+  class Quantizer;
+  Quantizer *quantizer_{nullptr};
+
+  friend class QuantizerTest;
```

Comment:
please remove `friend class QuantizerTest` in `analysis_predictor.h`. Header file doesn't need to contain a test class.

Reply:
The `Quantizer` class is a private nested class, and we need the friendship declaration to create an instance of `Quantizer` in the `QuantizerTest` test (`analysis_predictor_tester.cc`).

We could remove the declaration by moving the `class Quantizer;` forward declaration to the `public` section, but we would like to keep the `Quantizer` method hidden as an implementation detail and not to expose it to the outside.

Reply:
That's OK I think.

But the Quantize method should be general, or it should be named to `MkldnnQuantize`

Reply:
> #ifdef PADDLE_WITH_TESTING, be named to MkldnnQuantize

I agree with it @wojtuss @Superjomn

Reply:
@luotao1 , done.

Reply:
MkldnnQuantizerTest

Reply:
@luotao1 , @Superjomn , to avoid too many updates of the PR we could agree to add the `Mkldnn` prefix to all "quantize"-related methods, classes, and file names in this PR. What do you think?

Reply:
I think it is OK.

Reply:
>  all "quantize"-related methods, classes, and file names

Do you mean MkldnnQuantizerTest, EnableMkldnnQuantizer, mkldnn_quantizer.cc,mkldnn_ quantizer_config.cc, use_mkldnn_quantizer_, mkldnn_quantizer_config_, MkldnnQuantizer, mkldnn_quantizer_enabled,etc?

Reply:
yes

Reply:
I think it is OK.

Reply:
@luotao1, done.